README.md

```diff
@@ -110,7 +110,7 @@ def mystery6(s)
 end
 ```
 
-- What is mystery6("goodnight moon")?
-- What is mystery6("Ada Developers Academy")?
-- What is mystery6("Hi, there!")?
+- What is mystery6("goodnight moon")? "moon goodnight"
+- What is mystery6("Ada Developers Academy")? "academy developers ada"
+- What is mystery6("Hi, there!")? "there! hi"
```

Comment:
Minor error, but notable: there should be a leading space on each of the answers " moon goodnight", " academy developers ada", " there! hi,"